### ISSUE TITLE
Wrap long attributes and text in a heredoc

### ISSUE THREAD

**Issue by paulcsmith** [luckyframework/html2lucky#17]
So if you have a huge list of classes or text, we should try to use a heredoc to make it more manageable

I'm not sure how exactly this would work or how complicated it would be, but I think it'd be super handy

**Comment by edwardloveall** [comment]
I dig it. Could you put an example in here so when someone gets around to this they have something easy to refer to as a goal?

**Comment by paulcsmith** [comment]
@edwardloveall Yeah I will try to do this, but haven't actually done it so not sure how/if it'd work.

But I think it'd be something like this:

```crystal
a "Some text", 
  class: <<-CSS.lines.join(" ")
    lots of classes lots of classes 
    that are split at 80 chars
    CSS,
  other_attribute: "foo"
```

^ I'm not sure if this works, but I think it will

Text is easier:

```crystal
para do
  text <<-TEXT.lines.join(" ")
    Long text
    split at 80 chars
    TEXT
end
```
